Game/ui/asteroid/cellular_player_board.gd

```diff
@@ -33,32 +37,58 @@ func _ready() -> void:
 	player_name_label.text = "%s\n(%s)" % [player.name, player.index]
 
 	factory_and_mine.custom_minimum_size = Vector2i(0, layer_height_px * num_layers)
-	player_tile_map.tile_map_scale = WorldGenModel.tile_map_scale
-	player_tile_map.layer_thickness = WorldGenModel.layer_thickness
+	
+	game_grid.generate_grid(WorldGenModel.layer_thickness*(WorldGenModel.get_num_mine_layers() + 1), WorldGenModel.num_cols)
```

Comment:
`WorldGenModel.layer_thickness*(WorldGenModel.get_num_mine_layers() + 1)` should probably be extracted to a function in WorldGenModel. I know we already have `get_total_num_layers()` which returns the total number of layers including the factory layer, but nothing for the total number of rows.